Fix the expected return value of authentication plugins "verify" function (#271)
kuzzleio/documentation: src/api/1/controller-collection/create/index.md

{{{since "1.3.0"}}}

> You can also provide an optional body with a [collection mapping]({{ site_base_path }}guide/1/essentials/database-mappings) that allow you to exploit the full capabilities of our persistent data storage layer.

Reviewer: ```suggestion
You can also provide an optional body with a [collection mapping]({{ site_base_path }}guide/1/essentials/database-mappings) allowing you to exploit the full capabilities of our persistent data storage layer.
```

(or `that allows you`)

```js
{

>   "dynamic": "false",

Reviewer: ```suggestion
  "dynamic": "[false|true|strict]",
```

  "controller": "collection",
  "action": "create",
  "body": {

>     "dynamic": "false",

Reviewer: ```suggestion
    "dynamic": "[false|true|strict]",
```

## Arguments

* `collection`: name of the collection to create
* `index`: index that will host the new collection

---

## Body properties

### Optional:

> * `dynamic`: [dynamic mapping policy]({{ site_base_path}}guide/1/essentials/database-mappings/#dynamic-mapping-policy) for new fields

Reviewer: You should list the allowing values and the default:

```suggestion
* `dynamic`: [dynamic mapping policy]({{ site_base_path}}guide/1/essentials/database-mappings/#dynamic-mapping-policy) for new fields. Allowed values: `true` (default), `false`, `strict`
```
